.github/ISSUE_TEMPLATE/setup.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: Setup
+about: Prepare for development with basic project setup.
+title: '[ prep ] '
+labels: ''
+assignees: ''
+
+---
+
+Please check off line-items as they are completed and leave notes if necessary.
+If an item is not relevant to this project, [strike it out](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#styling-text)
+(e.g. `~~Not relevant item~~`) or remove it. If child tickets are created for
+any line-item, please update this ticket description to include references to
+those tickets.
+
+- [ ] Create a README.
+- [ ] Set up [Tugboat](https://dashboard.tugboat.qa/).
+- [ ] Set up Dependabot ([example](https://github.com/ChromaticHQ/chromatichq.com/blob/master/.github/dependabot.yml)).
+- [ ] Set up PHP code style checks.
+- [ ] Set up front-end linting.
```

Comment:
This is part of the “Set up front-end tooling” step.

Reply:
Removing. Thanks for calling this out and adding the fantastic suggestion!